src/cpp/runner/runnerapp.h

```diff
@@ -31,12 +31,10 @@ using std::map;
 using SignalInt = boost::signals2::signal<void(int)>;
 using Connection = boost::signals2::scoped_connection;
 
-class RunnerApp : public QObject
+class RunnerApp
 {
-	Q_OBJECT
-
 public:
-	RunnerApp(QObject *parent = 0);
+	RunnerApp();
 	~RunnerApp();
```

Comment:
can remove this declaration

Reply:
we need to keep destructor def due to :

function `RunnerAppMain::app_quit(int)':
         runnermain.cpp:41: reference to `RunnerApp::~RunnerApp()'